src/features/MentorPage/ListCard/ListCardHeader.tsx

```diff
@@ -0,0 +1,152 @@
+import styled from 'styled-components';
+import * as cssVariables from '../../../components/variables';
+import ProfilePicPlaceholder from '../../../static/img/icon-chat-profilepic.svg';
+import CSS from 'csstype';
+
+type Props = {
+  name: string;
+  age: number;
+  region: string;
+  available: boolean;
+  newMentor: boolean;
+  message: string;
+};
+
+const customStylus: CSS.Properties = {
+  margin: '0px',
+  display: 'flex',
+  flexWrap: 'wrap',
+};
+
+const truncateText: CSS.Properties = {
+  textOverflow: 'ellipsis',
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  margin: '0px',
+  width: '100%',
+};
+
+const ListCardHeader: React.FC<Props> = ({
+  name,
+  age,
+  region,
+  available,
+  newMentor,
+  message,
+}) => {
+  let availabilityMessage = newMentor ? 'Uusi' : '';
+  availabilityMessage = available ? availabilityMessage : 'Ei tavoitettavissa';
+  return (
+    <StyledListCardHeader isSelected={available}>
+      <MentorAvailability
+        isShowing={availabilityMessage != ''}
+        isNew={availabilityMessage === 'Uusi'}
+      >
+        {availabilityMessage}
+      </MentorAvailability>
+      <ProfilePicContainer />
+      <BasicInfoContainer>
+        <cssVariables.heading2_white style={{ margin: 0 }}>
+          {name}
+        </cssVariables.heading2_white>
+        <cssVariables.paragraph_white style={customStylus}>
+          {age} v. <StyledDivider>|</StyledDivider> {region}
+        </cssVariables.paragraph_white>
+        <cssVariables.paragraph_white style={{ margin: 0 }}>
+          {available}
+        </cssVariables.paragraph_white>
+        <cssVariables.paragraph_white title={message} style={truncateText}>
+          {message}
+        </cssVariables.paragraph_white>
```

Comment:
We talked yesterday about these <cssVariables.components>

I suggest following kind of approach where we set a "variant" to a Text component and then the styles come from there. Later on we can also enforce translated-strings with this component, if we type the translation-keys and we have translationId as a required-property =p

I am not 100% sure this is the best solution (in fact Im sure its not the best), because we cant now use styled from 'styled-components' to overwrite styles. But something like this.. I can try to come up with a better solution for this later too.. =P

But this can also be used, and then we can just use <Text> -component in the application everywhere. 
Maybe we dont need to use styled-components and instead we are happy with the styles-prop which is provided.

![Screenshot 2022-08-03 at 14 22 17](https://user-images.githubusercontent.com/34128180/182597245-962de9a9-c55e-46e3-a353-32cb0f7d1159.png)